frameworks/helloworld/tests/test_taskcfg.py

```diff
@@ -20,7 +20,7 @@ def configure_package(configure_security):
             config.SERVICE_NAME,
             0,
             {"service": {"yaml": "taskcfg"}},
-            wait_for_deployment=False,
```

Comment:
Could just remove the field since I think true is the default, but it's not worth running CI a second time over if it's green